aea/connections/stub/connection.py

```diff
@@ -240,12 +239,14 @@ def _split_messages(cls, data: bytes) -> List[bytes]:
 
     async def receive(self, *args, **kwargs) -> Optional["Envelope"]:
         """Receive an envelope."""
+        if self.in_queue is None:  # pragma: nocover
+            logger.error("Input queue not initialized.")
+            return None
+
         try:
-            assert self.in_queue is not None, "Input queue not initialized."
-            envelope = await self.in_queue.get()
-            return envelope
-        except Exception as e:
-            logger.exception(e)
+            return await self.in_queue.get()
+        except Exception:  # pylint: disable=broad-except
+            logger.exception("Stub connection receive error:")
```

Comment:
do we not need the `as e`? i,e, does `logger.exception` contain the `e`?

Reply:
it works with last exception